examples/js/exporters/GLTFExporter.js

```diff
@@ -1,5 +1,7 @@
 /**
  * @author fernandojsg / http://fernandojsg.com
+ * @author Don McCurdy / https://www.donmccurdy.com
```

Comment:
Shouldn't it be `donmccurdy` and `takahirox`?

Reply:
@mrdoob I just copied it from https://github.com/mrdoob/three.js/blob/dev/examples/js/loaders/GLTFLoader.js
@takahirox @donmccurdy any preference?

Reply:
I prefer Takahiro

Reply:
I tend to use the username to it's easier to find on github. But I don't have strong opinions about it.

Reply:
In my case, the link is to github account page, so easy to find hehe :D